Changelog.md

```diff
@@ -9,6 +9,7 @@ Compiler Features:
 
 Bugfixes:
  * Inline Assembly: Fix internal error when accessing incorrect constant variables.
+ * Reference Resolver: Fix internal error when accessing incorrect struct members.
```

Comment:
Is this really about struct members? Shouldn't it be more like "invalid user-defined type names"?